juju/juju
# WIP: Juju endpoint space display

## Please provide the following details to expedite Pull Request review:
## Description of change
Why is this change needed?
This PR has initial code changes required to display binding-space details in "juju status" command. this is targetting only for json and yaml formats.
## QA steps
How do we verify that the change works?
execute "juju status" command with --format json or yaml
## Documentation changes
Does it affect current user workflow? CLI? API?
n/a
## Bug reference
n/a
